ecl/hql/hqlir.cpp

```diff
@@ -2031,6 +2032,7 @@ id_t ExpressionIRPlayer::doProcessExpr(IHqlExpression * expr)
         {
             HqlExprArray scopeSymbols;
             expr->queryScope()->getSymbols(scopeSymbols);
+            scopeSymbols.sort(compareSymbolsByName);
```

Comment:
This has already been included in another PR - should really be rebasd on top of the current master